Sources/Basics/Triple+Basics.swift

```diff
@@ -116,6 +116,14 @@ extension Triple {
             )
         }
     }
+
+    public func matches(_ triple: Triple) -> Bool {
+        guard self.isMacOSX, let version = self._macOSVersion, let comparedVersion = triple._macOSVersion else {
```

Comment:
```suggestion
        guard self.isDarwin(), self.osNameUnversioned == triple.osNameUnversioned, let version = self.osVersion, let comparedVersion = triple.osVersion else {
```

I see that this is only used on the host triple atm (so assuming Darwin == macOS is reasonable) but calling it `matches(_:)` it's easy to envision it being used in a target-triple context in the future without knowing that it's macOS-specific. Alternatively, maybe the function should be renamed and/or made a fileprivate extension.

Reply:
I rewrote that check and inlined it in `SwiftSDKBundle`.